src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        File file = new File(fileName);
+        for (byte datas : data) {
+            try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(fileName,
```

Comment:
you open a new BufferedWriter for each byte - this is not effective

also, let's better use `FileOutputStream` here